dojo/tools/eslint/parser.py

```diff
@@ -0,0 +1,59 @@
+__author__ = 'aaronweaver'
+
+import json
+from dojo.models import Finding
+
+
+class ESLintParser(object):
+    def _convert_eslint_severity_to_dojo_severity(self, eslint_severity):
+        if eslint_severity == 2:
+            return "High"
+        elif eslint_severity == 1:
+            return "Medium"
+        else:
+            return None
```

Comment:
Is there never an opportunity for any vulnerable to have a severity of critical or lower than medium?

Reply:
Yep, ESLint support only error or warning.